frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1104,12 +1106,9 @@
 
         vm.starChallenge = function() {
             var parameters = {};
-            var phase_id = vm.phaseId;
-            var fileSelected = vm.fileSelected;
-            var url = stringToTemplate.convert(challenge.CHALLENGE.DOWNLOAD_ALL_SUBMISSIONS_ENDPOINT, {challenge_id: challenge_id});
-            var phaseurl = stringToTemplate.convert(url,{ phase_id: phase_id});
-            parameters.url = stringToTemplate.convert(phaseurl,{fileSelected: fileSelected});
-            parameters.method = "GET";
+            parameters.url = "challenges/"+ vm.challengeId + "/";
```

Comment:
This is the latest addition by someone else, will add the endpoint in config in next PR. @spyshiv

Reply:
cool